src/main/kotlin/com/github/avrokotlin/avro4k/Avro.kt

```diff
@@ -22,34 +23,39 @@ import java.nio.file.Paths
 
 open class AvroInputStreamBuilder<T>(
    private val converter: (Any) -> T
-   ) {
+) {
    /**
     * The format that should be used to decode the bytes from the input stream.
     */
-   var decodeFormat : AvroDecodeFormat = defaultDecodeFormat
+   var decodeFormat: AvroDecodeFormat = defaultDecodeFormat
+
```

Comment:
Could you deactivate your auto-format? While I generally like a consistent code style, unfortunately, there is currently no consistent code style in this project. Thus applying auto format only makes it harder to read PRs because of the additional diffs.

Reply:
I'll add the necessary to unify the formatting rules automatically into another pr. Until that, as said, please commit only your changes

Reply:
Will do. My autoformatting is set to respect the editorconfig settings, so this is a bit confusing to me ... but I will look into it.

Reply:
Yes but it is kind of empty, we will clean it in another pr. For the moment, if you can only commit the real changes even if it's in another format to make less noise for the review, that would be great 🙏

Reply:
done